resolwe_bio/processes/variant_calling/gatk.yml

```diff
@@ -67,17 +70,127 @@
     program: |
       SAMPLE_SLUG={{ alignment|sample_slug }}
 
-      java -jar /usr/gitc/GATK36.jar -T HaplotypeCaller -R {{genome.fasta.file}} -I {{alignment.bam.file}} -stand_call_conf {{stand_call_conf}} -stand_emit_conf {{stand_emit_conf}} -mbq {{mbq}} --dbsnp {{ dbsnp.vcf.file }} -L {{intervals.bedfile.file}} -o "${SAMPLE_SLUG}.gatkHC.vcf"
+      java -jar /usr/gitc/GATK36.jar \
+        -T HaplotypeCaller \
+        -R {{genome.fasta.file}} \
+        -I {{alignment.bam.file}} \
+        -stand_call_conf {{stand_call_conf}} \
+        -stand_emit_conf {{stand_emit_conf}} \
+        -mbq {{mbq}} \
+        --dbsnp {{ dbsnp.vcf.file }} \
+        -L {{intervals.bedfile.file}} \
+        -o "${SAMPLE_SLUG}.gatkHC.vcf"
+
       echo "{\"proc.progress\":0.80,\"proc.rc\":$?}"
 
-      /usr/gitc/bgzip -c "${SAMPLE_SLUG}.gatkHC.vcf" > "${SAMPLE_SLUG}.gatkHC.vcf.bgz"
+      /usr/gitc/bgzip -c "${SAMPLE_SLUG}.gatkHC.vcf" > "${SAMPLE_SLUG}.gatkHC.vcf.gz"
       echo "{\"proc.progress\":0.90,\"proc.rc\":$?}"
 
-      /usr/gitc/tabix -p vcf "${SAMPLE_SLUG}.gatkHC.vcf.bgz"
+      /usr/gitc/tabix -p vcf "${SAMPLE_SLUG}.gatkHC.vcf.gz"
       echo "{\"proc.progress\":0.95,\"proc.rc\":$?}"
 
-      echo "{\"vcf\":{\"file\":\"${SAMPLE_SLUG}.gatkHC.vcf\",\"refs\":[\"${SAMPLE_SLUG}.gatkHC.vcf.bgz\", \"${SAMPLE_SLUG}.gatkHC.vcf.bgz.tbi\"]}}"
+      echo "{\"vcf\":{\"file\":\"${SAMPLE_SLUG}.gatkHC.vcf.gz\"}}"
```

Comment:
`re-save file`?

Reply:
This process uses a third-party docker image, which does not include `resolwe runtime utils`